unity-renderer/Assets/UIComponents/Scripts/Components/Image/ImageComponentView.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public interface IImageComponentView
+{
+    /// <summary>
+    /// Fill the model and updates the image with this data.
+    /// </summary>
+    /// <param name="model">Data to configure the image.</param>
+    void Configure(ImageComponentModel model);
+
+    /// <summary>
+    /// Set an image.
+    /// </summary>
+    /// <param name="sprite">A sprite.</param>
+    void SetImage(Sprite sprite);
+
+    /// <summary>
+    /// Active or deactive the loading indicator.
+    /// </summary>
+    /// <param name="isVisible">True for showing the loading indicator and hidding the image.</param>
```

Comment:
```suggestion
    /// <param name="isVisible">True for showing the loading indicator and hiding the image.</param>
```

Reply:
Fixed in: https://github.com/decentraland/unity-renderer/pull/1175/commits/e8b08eea7973fffbf5d6bd5c38c02dc848f54050